students/johnpharmd/lesson10/fpmailroomtest.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+import mailroom4fp
+import os
+import datetime
+
+
+class MailroomTest(unittest.TestCase):
+    # maxDiff = None
+
+    def setUp(self):
+        self.donors_amts = mailroom4fp.get_donors_amts()
+
+    def test_donors(self):
+        for donor in self.donors_amts:
+            if '-' not in donor:
+                assert donor.isalpha()
+            else:
+                assert donor.replace('-', '').isalpha()
+
+    def test_titles(self):
+        for donor in self.donors_amts:
+            assert self.donors_amts[donor]['title'] in ('Prof.',
+                                                        'Dr.', 'Ms.', 'Mr.')
+
+    def test_donations(self):
+        for donor in self.donors_amts:
+            assert str(self.donors_amts[donor]['donations']).isdigit()
+
+    def test_num_of_donations(self):
+        for donor in self.donors_amts:
+            assert str(self.donors_amts[donor]['num_of_donations']).isdigit()
```

Comment:
Above tests are not necessary since they are not actually testing your code but what is in the dictionary.